### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by kbacha** [CircleCI-Public/browser-tools-orb#43]
### Orb version

```
  browser-tools: circleci/browser-tools@1.1.0
```

### What happened

Sometimes, we end up getting these errors when trying to install the chrome browser:

```
Google Chrome is not currently installed; installing it
curl: (18) transfer closed with 81823664 bytes remaining to read

Exited with code exit status 18

CircleCI received exit code 18
```

### Expected behavior

Chrome is installed.

**Comment by kbacha** [comment]
We're running on linux. I did browse the history and noticed that there was a commit that looked related. I've updated our version to 1.2.5

**Comment by HennaAbbas** [comment]
@kbacha can you confirm that updating to use version 1.2.5 resolved the issue?

**Comment by kbacha** [comment]
It’ll take a few builds to be confident. It wasn’t consistently failing

**Comment by seand7565** [comment]
@HennaAbbas Experiencing the same issue on 1.2.5 on linux:
<img width="373" alt="image" src="https://user-images.githubusercontent.com/5720486/161147701-b56021ea-54b3-4be6-9443-f2a75222bfdc.png">
<img width="263" alt="image" src="https://user-images.githubusercontent.com/5720486/161147963-23d64d20-a8e8-49b5-ad39-76967fb082af.png">

**Comment by davidwparker** [comment]
We get this a lot of the time now too.

```
orbs:
  ruby: circleci/ruby@0.1.2
  browser-tools: circleci/browser-tools@1.2.5
  cypress: cypress-io/cypress@1.26.0
```

**Comment by kbacha** [comment]
Ok, it's been a day and I've still seen the error after updating. It also appears I'm not alone.

**Comment by davidwparker** [comment]
@KyleTryon - this happens very consistently. Is there someone on the inside who knows what changed?

**Comment by abhinaba-ghosh** [comment]
Can confirm happening on Mac executor

**Comment by KyleTryon** [comment]
Hello all,

I have a PR open here that aims to resolve this issue. If anyone is interested, you can help us out by testing out this version of the orb before it goes live.

https://github.com/CircleCI-Public/browser-tools-orb/pull/48#issuecomment-1099241298

**Comment by KyleTryon** [comment]
Hello,

We have just released version 1.3 of the browser-tools orb with this issue in mind. Please try this version and let us know if you experience any issues.

https://github.com/CircleCI-Public/browser-tools-orb/releases/tag/v1.3.0
https://circleci.com/developer/orbs/orb/circleci/browser-tools?version=1.3.0
